### PR TITLE
Ps 4688 ssl certificate compatibility matrix 8.0

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
Please read my last comment on https://jira.percona.com/browse/PS-4688

**Comment by borys-belinsky** [comment]
Ready for review

File: doc/source/management/ssl-improvement.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.. _percona-server.management.ssl:
+
+================================================================================
+SSL Improvements
+================================================================================
+
+|Percona Server| enables transmitting data in the encrypted form by
```
Comment (laurynas-biveinis) [inline doc/source/management/ssl-improvement.rst]: Remove paragraph

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/management/ssl-improvement.rst (hunk 1 above)
Comment (borys-belinsky) [reply doc/source/management/ssl-improvement.rst]: Ready for review

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**